examples-copier/configs/copier-config-examples/copier-config-workflow.yaml

```diff
@@ -0,0 +1,236 @@
+      - move:
+          from: "mflix/server/js-express"
+          to: "server"
+      
+      # Copy README (renaming)
+      - copy:
+          from: "mflix/README-JAVASCRIPT-EXPRESS.md"
+          to: "README.md"
+      
+      # Copy .gitignore (renaming)
+      - copy:
+          from: "mflix/.gitignore-js"
+          to: ".gitignore"
+    
+    commit_strategy:
+      type: "pull_request"
+      pr_title: "Update MFlix application from docs-sample-apps"
+      pr_body: |
+        Automated update of MFlix Node.js application
+        
+        **Source Information:**
+        - Repository: mongodb/docs-sample-apps
+        - Branch: main
+        
+        **Changes:**
+        - Client files (Next.js)
+        - Server files (Node.js/Express)
+        - README and .gitignore
+        
+        This PR includes all synchronized files from the source repository.
+      auto_merge: false
+
+  # ==========================================================================
+  # Python MFlix: mongodb/docs-sample-apps → mongodb/sample-app-python-mflix
+  # ==========================================================================
+  
+  - name: "mflix-python"
+    source:
+      repo: "mongodb/docs-sample-apps"
+      branch: "main"
+    destination:
+      repo: "mongodb/sample-app-python-mflix"
+      branch: "main"
+    transformations:
+      # Copy client directory
+      - move:
+          from: "mflix/client"
+          to: "client"
+      
+      # Copy Python server (removing python-fastapi subdirectory)
+      - move:
+          from: "mflix/server/python-fastapi"
+          to: "server"
+      
+      # Copy README (renaming)
+      - copy:
+          from: "mflix/README-PYTHON-FASTAPI.md"
+          to: "README.md"
+      
+      # Copy .gitignore (renaming)
+      - copy:
+          from: "mflix/.gitignore-python"
+          to: ".gitignore"
+    
+    commit_strategy:
+      type: "pull_request"
+      pr_title: "Update MFlix application from docs-sample-apps"
+      pr_body: |
+        Automated update of MFlix Python application
+        
+        **Source Information:**
+        - Repository: mongodb/docs-sample-apps
+        - Branch: main
+        
+        **Changes:**
+        - Client files (Next.js)
+        - Server files (Python/FastAPI)
+        - README and .gitignore
+        
+        This PR includes all synchronized files from the source repository.
+      auto_merge: false
+
+# ============================================================================
+# BENEFITS OF THIS FORMAT
+# ============================================================================
+#
+# 1. SIMPLER: 3 workflows instead of 12 rules (70% reduction)
+# 2. CLEARER: Each workflow shows complete source → destination mapping
+# 3. MULTI-ORG READY: Each workflow can have different source repo
+# 4. EASIER TO MAINTAIN: Add new apps by adding one workflow
+# 5. BETTER DEFAULTS: Global defaults reduce repetition
+#
```

Comment:
I suggest removing this section since it's comparing to an implementation that effectively nobody has seen.
```suggestion
```